expenses/migrations/0001_initial.py

```diff
@@ -20,9 +20,34 @@ class Migration(migrations.Migration):
             fields=[
                 ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('description', models.TextField(default='', max_length=250)),
-                ('category', models.CharField(choices=[('Rent', 'Rent'), ('Mortgage', 'Mortgage'), ('Bills', 'Bills'), ('Transportation', 'Transportation'), ('Clothing', 'Clothing'), ('Healthcare', 'Healthcare'), ('Food', 'Food'), ('Insurance', 'Insurance'), ('Kids', 'Kids'), ('Culture', 'Culture'), ('Vacations', 'Vacations'), ('Other', 'Other')], default='Other', max_length=32)),
-                ('amount', models.IntegerField(blank=True, null=True, validators=[django.core.validators.MinValueValidator(0)])),
-                ('date', models.DateTimeField(default=django.utils.timezone.now)),
+                (
+                    'category',
+                    models.CharField(
```

Comment:
I think that line 25 can be on line 24.
The context is more understandable, at least for me

Reply:
This is what my formatter formatted. I think its understandable but anyway its a migration and generated automatically so this is not that critical.